internal/controller/nonadminrestore_controller_test.go

```diff
@@ -388,7 +418,7 @@ var _ = ginkgo.Describe("Test full reconcile loop of NonAdminRestore Controller"
 				},
 			},
 		}),
-		ginkgo.Entry("Should update NonAdminRestore until it invalidates and then delete it", nonAdminRestoreFullReconcileScenario{
+		ginkgo.Entry("Should not include queueInfo for the given backup as NonAdminBackup is not ready to be restored (BackingOff)", nonAdminRestoreFullReconcileScenario{
```

Comment:
would not update test name

the test scenario did not suffer great changes

This info can be added as a comment in line 449

Reply:
I made it more precise of what it does, so why not have it better if I've changed status check that affected this run?